ogr/services/github/pull_request.py

```diff
@@ -167,8 +167,8 @@ def __get_fork(fork_username: str, repo: _GithubRepository) -> _GithubRepository
         return forks[0]
 
     @staticmethod
-    def get(project: "ogr_github.GithubProject", id: int) -> "PullRequest":
-        pr = project.github_repo.get_pull(number=id)
+    def get(project: "ogr_github.GithubProject", pr_id: int) -> "PullRequest":
+        pr = project.github_repo.get_pull(number=pr_id)
```

Comment:
Hmm. Change of the API? Can we safely rename that?

Reply:
It's internal, I doubt it is used independently of the project. It would be PITA to use, since you need to provide project separately.